apps/amakrushi/src/components/App.tsx

```diff
@@ -51,8 +52,25 @@ const App: NextPage = () => {
 
   return (
     <>
```

Comment:
move this container to the main file so that it can be accessed throughout the app.